extensions/elytron-security-oauth2/pom.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-elytron-security-oauth2-parent</artifactId>
```

Comment:
As proposed after at the forum, please consider renaming the whole extension to `quarkus-elytron-realm-token` (sorry I did not think about it when suggesting a new extension). `oauth2` can send a mixed message, `oidc` itself is `oauth2` based and your extension would verify the OIDC tokens as well, possibly those issued by KC itself :-).

Reply:
OK.
I use the name `<name>Quarkus Security OAuth2 - Runtime</name>` and a directory `elytron-security-oauth2` does this also needs renaming ?

Reply:
And does the package name and all the classes needs also renaming ?

Reply:
@loicmathieu lets postpone the `-oauth2` renaming bit till the moment the PR is ready to go otherwise, may be it will stay after all, lets double check on it later

Reply:
And the configuration group also ?

Reply:
@sberyozkin I saw you comment after I rename the artifacst
I will revert them as I prefer to be coherent and decide to rename everything or nothing.